docs/classic-vs-vt.md

```diff
@@ -0,0 +1,81 @@
+
+### Direct Window Access
+
+On Windows, the exact window handle to the hosting window is available through the Console API surface. This allows a command-line utility to perform advanced window operations by reaching into the wide gamut of Win32 APIs permitted against a window handle and manipulate the window state, frame, icon, or other properties about the window. By contrast, on other platforms with virtual terminal sequences, there is a specific gamut of commands that can be performed against the window to do things like changing its size or displayed title and they must be done in the same band and under the same control as the remainder of the stream.
+
+As Windows has evolved, the security controls and restrictions on window handles have increased. Additionally, the nature and existence of an application-addressable window handle on any specific user interface element has evolved, especially with the increased support of device form factors and platforms. This makes direct window access to command-line applications fragile as the platform and experiences evolve.
+
+### Unicode
+
+The implicit standard for communication across platforms and the web is Unicode, specifically in the UTF-8 form. Of course, other encodings still exist, but generally speaking, when otherwise undefined, using UTF-8 is accepted as the appropriate default balance of portability, storage/transmission size, and breadth of expression required to support the world's languages and glyphs.
+
+The Windows console platform has and will continue to support all existing code pages and encodings, but we recommend UTF-8 for all forward looking development and also accept UTF-16 as an algorithmically-translatable alternative representation of the same information that is more compatible with Windows' historical Unicode philosophies extending from initial UCS-2 implementations.
+
+UTF-8 support in the console can be found via the _A_ variant of all Console APIs or the filesystem [**WriteFile**](https://msdn.microsoft.com/library/windows/desktop/aa365747) and [**ReadFile**](https://msdn.microsoft.com/library/windows/desktop/aa365467) methods against console handles after setting the codepage to `65001` or `CP_UTF8` with the [**SetConsoleOutputCP**](setconsoleoutputcp.md) and [**SetConsoleCP**](setconsolecp.md) methods as appropriate. Setting the code pages in advance is only necessary if the machine has not chosen "Use Unicode UTF-8 for worldwide language support" in the settings for Non-Unicode applications in the Region section of the Control Panel.

```

Comment:
We've received reports () that ReadFile/ReadConsole doesn't work right in UTF-8 (https://github.com/microsoft/terminal/issues/4551) in all cases. Should we comport implementation before making a suggestion? Does it not matter b/c COOKED?

Reply:
I ripped it. We can suggest it should we ever make it work.